Assets/HoloToolkit/SpatialSound/Scripts/AudioOccluder.cs

```diff
@@ -9,47 +9,62 @@ namespace HoloToolkit.Unity
+        IAudioInfluencer
     {
         /// <summary>
-        /// Frequency above the nominal range of human hearing. Applying this frequency to the filter will have no perceived impact on the audio source.
+        /// Frequency above which sound will not be heard after applying occlusion.
+        /// Setting this value to 22000.0 effectively disables the effect.
         /// </summary>
-        private readonly float NeutralFrequency = 22000f;
-
-        [Tooltip("Frequency above which sound will not be heard.")]
+        /// <remarks>
+        /// Chaining occluders will result in the lowest of the cutoff frequencies being applied to the sound.
+        /// The CutoffFrequency range is 0.0 - 22000.0 (0 - 22kHz), inclusive.
+        /// The default value is 5000.0 (5kHz).
+        /// </remarks>
+        [Tooltip("Frequency above which sound will not be heard after applying occlusion.")]
         [Range(10.0f, 22000.0f)]
         public float CutoffFrequency = 5000.0f;
-        
+
+        /// <summary>
+        /// Percentage of the audio source volume that will be heard after applying occlusion.
+        /// </summary>
+        /// <remarks>
+        /// VolumePassThrough is cumulative. It is applied to the current volume of the object at the time
+        /// the effect is applied.
+        /// The VolumePassThrough range is from 0.0 - 1.0 (0-100%), inclusive.
+        /// The default value is 1.0.
+        /// </remarks>
         [Tooltip("Percentage of the audio source volume that will be heard after applying occlusion.")]
         [Range(0.0f, 1.0f)]
         public float VolumePassThrough = 1.0f;
 
-        // Update is not used, but is kept so that this component can be enabled/disabled.
-        private void Update() 
-        { }
-
         /// <summary>
         /// Applies the audio effect.
         /// </summary>
-        /// <param name="emitter">The GameObject on which the effect is to be applied.</param>
-        /// <param name="audioSource">The AudioSource that will be impacted by the effect.</param>
-        public void ApplyEffect(GameObject emitter,
-                                AudioSource audioSource)
+        /// <param name="soundEmittingObject">The GameObject on which the effect is to be applied.</param>
+        public void ApplyEffect(GameObject soundEmittingObject)
         {
             if (!isActiveAndEnabled)
             { return; }
 
+            AudioSource audioSource = soundEmittingObject.GetComponent<AudioSource>();
             if (audioSource == null)
             {
                 Debug.LogWarning("The specified emitter does not have an attached AudioSource component.");
                 return;
             }
 
             // Audio occlusion is performed using a low pass filter.                
-            AudioLowPassFilter lowPass = emitter.GetComponent<AudioLowPassFilter>();
+            AudioLowPassFilter lowPass = soundEmittingObject.GetComponent<AudioLowPassFilter>();
```

Comment:
What was the purpose of renaming this? Just for cosmetic reasons?
Could we revert these just so it's a bit easier to focus on the real changes?

Reply:
This was changed for clarity.